From 64f3eba6ffe3fcc12a4a6ea85420a829114f89f0 Mon Sep 17 00:00:00 2001
From: Tim McGilchrist <timmcgil@gmail.com>
Date: Thu, 18 Oct 2018 17:59:32 +1100
Subject: [PATCH] Fixup deprecation warning about for function.

    src/Action/Server.hs:206:49: warning: [-Wdeprecations]
    In the use of for (imported from Data.List.Extra):
    Deprecated: Use flip map directly, since this function clashes with Data.Traversable.for
---
 src/Action/Server.hs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/Action/Server.hs
+++ b/src/Action/Server.hs
@@ -203,7 +203,7 @@ itemCategories xs =
     nubOrd [("package",strUnpack p) | Just (p,_) <- map targetPackage xs]
 
 showFroms :: Bool -> Maybe FilePath -> [Target] -> String
-showFroms local haddock xs = intercalate ", " $ for pkgs $ \p ->
+showFroms local haddock xs = intercalate ", " $ (flip map) pkgs $ \p ->
     let ms = filter ((==) p . targetPackage) xs
     in unwords ["<a href=\"" ++ showURL local haddock b ++ "\">" ++ strUnpack a ++ "</a>" | (a,b) <- catMaybes $ p : map remod ms]
     where
-- 
2.26.0

